grpc/channelz/v1/channelz.proto

```diff
@@ -435,9 +436,14 @@ service Channelz {
 }
 
 message GetTopChannelsRequest {
```

Comment:
I don't think `GetTopChannelRequest` really need the limit, as we also have the `GetChannel` API.